front/components/poke/shadcn/ui/data_table.tsx

```diff
@@ -0,0 +1,119 @@
+"use client";
```

Comment:
I don't think we should have this ? This only makes sense if we have RSC

Reply:
Let me remove it.